### PR TITLE
Essay caption

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Adds support for the version ofthe caption that appears on Photo Essays

Screenshot 2020-06-17 at 09 24 03
Screenshot 2020-06-17 at 09 23 30

This PR includes the following changes:

1. `designType` is passed down the stack into `Caption`
2. The switch on designType pattern is added to `Caption`
3. Branched styling for `PhotoEssay` has been added
4. `AticleBody` was edited to remove the global css selector on `figcaption`

### CODE REVIEW

File: src/web/components/Caption.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    display: Display;
+    designType: DesignType;
+    captionText?: string;
+    pillar: Pillar;
+    padCaption?: boolean;
+    credit?: string;
+    displayCredit?: boolean;
+    shouldLimitWidth?: boolean;
+    isOverlayed?: boolean; // Not tested here as this option only works in the context of the ImageComponent
+ */
```
Comment (liywjl) [inline src/web/components/Caption.stories.tsx:21]: to remove

Comment (oliverlloyd) [reply src/web/components/Caption.stories.tsx:21]: Why? I often paste the props in the stories as comments so the reader has an immediate reference to help understand the scope of testing

Comment (liywjl) [reply src/web/components/Caption.stories.tsx:21]: dont you get that from typescript anyway by simply hover over the component?
ex:
Screenshot 2020-06-17 at 10 13 33

Comment (oliverlloyd) [reply src/web/components/Caption.stories.tsx:21]: Sure, there are helper tooltips. I just find it easier to see it on the page rather then having to reveal it by moving my mouse. I guess the real question is: why not?

Comment (liywjl) [reply src/web/components/Caption.stories.tsx:21]: > why not?
sure ^^

**Comment by oliverlloyd** [comment]
[Support `PhotoEssay` version of Caption](https://trello.com/c/OfE2TCec/1636-support-photoessay-version-of-caption)

**Pull request merged**